encoding/basics-and-workflow.md

```diff
@@ -2,36 +2,584 @@
+over the AAC encoder supplied with ffmpeg.
+It does a much better job of preserving the perceptional quality
+over all its competitors,
+and is thus the go-to for people that encode in AAC.
+
+The options you'll used most is `-V`.
+`-V` sets the true Variable Bitrate mode,
+allowing qaac free reign in deciding
+what it believes to be the best bitrate to encode to.
+
+The most common command looks like this:
+
+`$ qaac input.wav -V 127`
+
+Let's compare the filesizes between the FLAC encodes we did before.
+
+```
+26.811.464 Fate_Zero_OP2_(original).wav
+19.469.016 Fate_Zero_OP2_(12)_24bit.flac
+10.898.384 Fate_Zero_OP2_(12)_16bit.flac
+ 3.886.831 Fate_Zero_OP2_(qaac).m4a
+```
+```
+458.233.964 KaguyaBD_01_(original).wav
+297.952.201 KaguyaBD_01_(12)_24bit.flac
+147.327.448 KaguyaBD_01_(12)_16bit.flac
+ 54.191.826 KaguyaBD_01_(qaac).m4a
+```
+
+The final commonly-used audio codec is opus.
+Opus is used for web audio,
+as it is designed for
+low-latency, low-complexity audio compression.
+This makes it great for "mini" releases.
+You can encode to qaac through ffmpeg:
+
+`$ ffmpeg -i input.wav output.opus`
+
+Or through [Opus-tools](opus-tools)
+if you don't know
+how to compile ffmpeg with opus support:
+
+`$ opusenc input.wav output.opus`
+
+Unlike qaac, opus works with a constant bitrate.
+You can set it with `-b:a` in ffmpeg
+and `--bitrate` in opusenc.
+For anime, a bitrate of 128k is usually transparent
+for both dialogue and music.
+
+`$ opusenc input.wav --bitrate 128k output.opus`
 
 ## Muxing
 
-mkvtoolnix
+The most common tool for muxing is [mkvtoolnix](mkvtoolnix).
+It offers an easy-to-use GUI,
+as well as CLI for those that prefer that.
+
+The input tab is where you drag most of your components.
+This can be video, audio, and subtitles.
+You can also set the languages
+and additional settings for every track,
+like for example delays, track names, playback fps, etc.
+
+![mkvtoolnix Input tab](images/basics_mkvtoolnix_input.png)
+
+The output tab allows you to change the output.
+You can for example trim the output,
+set the segments, file title, etc.
+This is also where you load in the chapters
+if your video has any.
+
+![mkvtoolnix Output tab](images/basics_mkvtoolnix_output.png)
+
+The attachments tab is where you add attachments
+to the output video.
+These are generally fonts.
+
+![mkvtoolnix Attachments tab](images/basics_mkvtoolnix_attachments.png)
+
+The Chapter Editor allows you to create and edit chapters.
+These are used to skip to certain parts of the video,
+like for example the Opening.
+
+![mkvtoolnix Chapters page](images/basics_mkvtoolnix_chapter.png)
+
+For CLI, you can do a simple mux with the following command:
+
+`$ mkvmerge -o output.mkv input_video.264 input_audio.m4a`
+
+For a list of all settings,
+check the [documentation](mkvmerge_docs).
+
+***
+
+[^1]: "funcs" are a combination of wrappers written by people to perform different tasks.
+Most of these can be found in the [VapourSynth Database][vsdb]
```

Comment:
Iirc you cannot use linebreaks in footnotes. @OrangeChannel?

Reply:
Yea, that's correct.